test/odbc/drivers.h

```diff
@@ -0,0 +1,30 @@
+#ifndef DRIVERS_H
+#define DRIVERS_H
+
+#include "constants.h"
+#include "connection_string_object.h"
+
+using namespace constants;
+
+class Drivers {
+
+  public:
+
+    // Constructor
+    Drivers();
+
+    // Destructor
+    ~Drivers();
+
+    void SetDrivers();
+
+    map<ServerType, ConnectionStringObject> getOdbcDrivers();
+
+  private:
+
+    map<ServerType, ConnectionStringObject> odbc_drivers{};
```

Comment:
nit: can we name this `odbc_drivers_` to follow member variable naming convention?

Reply:
Done.